demo/src/html/index.html

```diff
@@ -3,22 +3,14 @@
     <head>
         <meta charset="UTF-8" />
         <title><%= htmlWebpackPlugin.options.title %></title>
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
In general, I prefer not loading external fonts. In this case I'm fine with the existing fonts, so you should revert these changes.